### DISCUSSION TITLE
How to use Clerk proxy from localhost?

### DISCUSSION THREAD

**Discussion by lancejpollard** [clerk/javascript#3080] [Support request - DO NOT OPEN HERE]
Having mostly resolved https://github.com/orgs/clerk/discussions/3076 now (see my answer [here](https://stackoverflow.com/a/78254667/169992)), now I am currently facing this error when loading the Next.js locally using the `<ClerkProvider proxyUrl="https://example.com/proxy">` React provider:

> Access to fetch at 'https://example.com/proxy/v1/dev_browser?_clerk_js_version=4.70.6' from origin 'http://localhost:3000' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header has a value 'https://example.com' that is not equal to the supplied origin. Have the server send the header with a valid value, or, if an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

How do I get this to work from localhost? If I change it to `<ClerkProvider proxyUrl="http://localhost:3000/proxy">`, it just breaks:

```
GET https://localhost:3000/bind/npm/@clerk/clerk-js@4/dist/clerk.browser.js net::ERR_SSL_PROTOCOL_ERROR        
Error: Clerk: Failed to load Clerk
    at eval (loadClerkJsScript.js:20:1)
    at async _IsomorphicClerk.loadClerkJS (isomorphicClerk.js:587:1)
```

**Comment by colevr1** [comment]
they hardcoded https in their config inside node modules.

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
